src/lib/GOBLBuilder.svelte

```diff
@@ -1,15 +1,36 @@
 <script lang="ts">
-  import MenuBar from "./ui/MenuBar.svelte";
-  import Toolbar from "./toolbar/Toolbar.svelte";
+  import { editor, envelope } from "$lib/stores.js";
+  import MenuBar from "./menubar/MenuBar.svelte";
   import Editor from "./editor/Editor.svelte";
 
-  export let jsonSchemaURL: string;
+  export let jsonSchemaURL = "";
+  export let value = "";
```

Comment:
Perhaps `data` would be a better name? Value tends to imply it's a short value. It could be `envelope` or `document` also, but those are maybe too specific.

Reply:
> Perhaps `data` would be a better name? Value tends to imply it's a short value. It could be `envelope` or `document` also, but those are maybe too specific.

Initially went with `value` because that's the property name that Monaco uses, but since our component entails more than just the editor, I think `data` is indeed a good alternative 👍

Reply:
Oh, that makes sense! Wouldn't have expected that from Monaco. Perhaps it comes from the HTML tags which typically have a `value` property. Doesn't apply to `<textarea>` though... `data` is cool 👍